src/main/kotlin/step2/enums/LottoReturn.kt

```diff
@@ -0,0 +1,18 @@
+package step2.enums
+
+enum class LottoReturn(
+    val matchCount: Int,
+    val returnPrice: Int,
+) {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
코틀린에서는 언더바를 사용하면 천단위 숫자를 좀 더 가독성있게 작성할 수 있어요